subsys/net/ip/tp.c

```diff
@@ -8,6 +8,8 @@
 #include <stdlib.h>
 #include "tp.h"
 #include "tp_priv.h"
+#include "ipv4.h"
```

Comment:
Could you split your patch in 2? tp is not tcp2.

Reply:
done